### PR TITLE
Adds a sprite for the donutgiver

### PR DESCRIPTION
**Pull request by Blithering**
Adds a sprite for the donutgiver
secpowercreep

:cl:
 * imageadd: Donutgiver now has its own icon

### CODE REVIEW

**Comment by drawsstuff** [comment]
i am going to bully any officer that takes this for being FAT

File: code/game/objects/items/trader.dm [hunk 1]
```diff
@@ -230,8 +230,8 @@ var/global/list/alcatraz_stuff = list(
 /obj/item/clothing/head/helmet/donutgiver
 	name = "donutgiver"
 	desc = "The Donutgiver III. A twenty-five sprinkle headgear with mission-variable voice-programmed confections. It has the words SPRINKLE, JELLY, CHAOS and FAVORITE etched onto its sides."
-	icon_state = "helmet_sec"
-	item_state = "helmet"
+	icon_state = "donutgiver"
+	item_state = "helmet_donuts"
 	species_fit = list(INSECT_SHAPED)
```
Comment (PlausibleDeniability) [inline code/game/objects/items/trader.dm:235]: I made an insectoid variant for this when it used the default helmet sprite, I'm not sure what'll occur if a bug wears this, will it just be invisible on bugs?

If so, you'll have to add one to the bug hats.dmi

Comment (Blithering) [reply code/game/objects/items/trader.dm:235]: The donutgiver refuses to be worn by nonhumans, did you change that part for insectoids?

Comment (PlausibleDeniability) [reply code/game/objects/items/trader.dm:235]: Ah shit, nah it's still restricted, stoopid racists

Comment (Blithering) [reply code/game/objects/items/trader.dm:235]: Racism is the only thing stopping the vox from keeping the donut helmets to themselves!

**Pull request merged**